pkg/kubectl/run.go

```diff
@@ -77,6 +80,20 @@ func (BasicReplicationController) Generate(genericParams map[string]interface{})
 		}
 		delete(genericParams, "args")
 	}
+
+	var envs []api.EnvVar
+	envStrings, found := genericParams["env"]
+	if found {
+		if envStringArray, isArray := envStrings.([]string); isArray {
```

Comment:
Follow the same pattern as above and return an error in case `envStrings` is not an array.